test/e2e/windows/security_context.go

```diff
@@ -71,6 +71,24 @@ var _ = SIGDescribe("[Feature:Windows] SecurityContext RunAsUserName", func() {
 		f.TestContainerOutput("check overridden username", pod, 0, []string{"ContainerUser"})
 		f.TestContainerOutput("check pod SecurityContext username", pod, 1, []string{"ContainerAdministrator"})
 	})
+	ginkgo.It("should ignore Linux Specific SecurityContext if set", func() {
+		ginkgo.By("Creating a pod with SELinux options")
+
+		windowsPodWithSELinux := createTestPod(f, "mcr.microsoft.com/powershell:lts-nanoserver-1809", windowsOS)
```

Comment:
I started with `agnhost` host image on Kubernetes cluster and it is working fine, however since I made this test distro specific(particularly SELinux), I tested with `OpenShift cluster` with Windows 1809 nodes, I noticed that the `agnhost` image pulling failed on Windows 1809 nodes. I can revert the change and test it downstream and get back to you later.